spec/integration/edge_gateway/data/static_routing_empty.yaml.mustache

```diff
@@ -0,0 +1,4 @@
+---
+gateway: {{ edge_gateway_name }}
+static_routing_service:
+	enabled: true
```

Comment:
Can you please fix the indenting on line 4 so that `enabled: true` is indented by two spaces?